crates/router/src/core/webhooks/incoming.rs

```diff
@@ -844,28 +842,17 @@ async fn payouts_incoming_webhook_flow(
 
         // If event is NOT an UnsupportedEvent, trigger Outgoing Webhook
         if let Some(outgoing_event_type) = event_type {
-            let router_response =
-                payouts::response_handler(&state, &merchant_context, &payout_data).await?;
-
-            let payout_create_response: payout_models::PayoutCreateResponse = match router_response
-            {
-                services::ApplicationResponse::Json(response) => response,
-                _ => Err(errors::ApiErrorResponse::WebhookResourceNotFound)
-                    .attach_printable("Failed to fetch the payout create response")?,
-            };
-
-            Box::pin(super::create_event_and_trigger_outgoing_webhook(
+            Box::pin(super::add_bulk_outgoing_webhook_task_to_process_tracker(
                 state,
-                merchant_context,
-                business_profile,
+                &business_profile,
+                &payout_attempt.payout_id,
                 outgoing_event_type,
-                enums::EventClass::Payouts,
-                updated_payout_attempt.payout_id.clone(),
-                enums::EventObjectType::PayoutDetails,
-                api::OutgoingWebhookContent::PayoutDetails(Box::new(payout_create_response)),
-                Some(updated_payout_attempt.created_at),
+                diesel_models::enums::EventClass::Payouts,
+                diesel_models::enums::EventObjectType::PayoutDetails,
+                Some(payout_attempt.created_at),
             ))
-            .await?;
+            .await
+            .map_err(|e| e.change_context(errors::ApiErrorResponse::WebhookProcessingFailure))?;
```

Comment:
Same comments as before, about the `enums::` qualification and the unnecessary `map_err()`.